modules/bbp-simulation/src/main/resources/schemas/hbphackaton/simulation/emodelgeneration/v0.1.0.json

```diff
@@ -0,0 +1,47 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/core/schema/v0.1.0",
+    {
+      "nsg": "https://bbp-nexus.epfl.ch/vocabs/bbp/neurosciencegraph/core/v0.1.0/",
+      "this": "{{base}}/schemas/hbphackaton/simulation/emodelgeneration/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/nexus/prov/generation/v0.1.0",
+    "{{base}}/schemas/bbp/core/quantitativevalue/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:EModelGenerationShape",
+      "@type": "sh:NodeShape",
+      "label": "Trace generation shape definition",
+      "targetClass": "nsg:TraceGeneration",
```

Comment:
When we discussed this, I had forgotten that we already have a type **nsg:TraceGeneration** for the qualified generation of biological traces. We should find an alternative here.